exact.go

```diff
@@ -0,0 +1,106 @@
+// Package exact implements common math for rational numbers.
+// You must use this package if you need to avoid floating point
+// roudoff errors.
+package exact
+
+type (
+	// Frac is a fraction
```

Comment:
It would be nice to have a fraction package only with fraction stuff. So the struct would be just F. Like fraction.F. Or perhaps fract.F (I'm more lenient with long names =P).

Reply:
I liked it =)
Then, the usage would be something like:

```go
fmt.Println(exact.Sqrt(fraction.F(1, 2))
```

Reply:
Other types other than fraction in a lib for exact computations?

Reply:
I see =P. Well there will be at least the big fractions and the small fractions no ?